app/experimenter/nimbus-ui/src/components/AppLayoutWithExperiment/index.test.tsx

```diff
@@ -146,34 +112,34 @@ describe("AppLayoutWithExperiment", () => {
       });
     });
   });
-
-  // TODO: EXP-733 some sort of after test cleanup, can't add tests after without errors
-  it("renders loading screen", () => {
-    render(<Subject />);
-    expect(screen.getByTestId("page-loading")).toBeInTheDocument();
-  });
 });
 
+jest.useFakeTimers();
+
+jest.mock("@reach/router", () => ({
+  ...jest.requireActual("@reach/router"),
+  navigate: jest.fn(),
+}));
+
 const Subject = ({
   mocks = [],
-  polling = false,
-  sidebar = true,
-  withTitle = true,
-  redirect = () => {},
```

Comment:
Scratched my head a little at these defaults going away, but looks like this all better matches the actual type of the tested component so :+1: